dashboard-generator/generator.js

```diff
@@ -4,7 +4,7 @@
 
 // utility functions
 var $ = (selector) => document.querySelector(selector);
-
+var countDash=0; //a variable to keep count of the added dashboards
```

Comment:
The style in this file is to put spaces around the equals sign in assignments, so it would be appropriate to include them here, too.